fulp_modules/main_features/bloodsuckers/code/bloodsucker/bloodsucker_flaws.dm

```diff
@@ -126,21 +129,21 @@
 				* Additionally, you magically protect your Vassals from being disconnected with you via Mindshielding, and can mutilate them by putting them on a persuasion rack.<br> \
 				* Finally, you can revive dead non-Vassals by using the Persuasion Rack as they lie on it.</span>")
 			return
-		if(CLAN_VENTRUE) // WILLARD TODO: Make a Ventrue-unique objective to drink X amount of Blood?
+		if(CLAN_VENTRUE)
 			my_clan = CLAN_VENTRUE
 			to_chat(owner, "<span class='announce'>You have Ranked up enough to learn: You are part of the Ventrue Clan!<br> \
 				* As part of the Ventrue Clan, you are extremely snobby with your meals, and refuse to drink blood from people without a Mind.<br> \
 				* Additionally, you will no longer Rank up. You are now instead able to get a Favorite vassal, by putting a Vassal on the persuasion rack and attempting to Tortute them.<br> \
-				* Finally, you may Rank your Favorite Vassal up by buckling them onto a Candelabrum.</span>")
+				* Finally, you may Rank your Favorite Vassal (and your own powers) up by buckling them onto a Candelabrum and using it, this will cost a Rank or Blood to do.</span>")
 			to_chat(owner, "<span class='announce'>* Bloodsucker Tip: Examine the Persuasion Rack/Candelabrum to see how they operate!</span>")
 			return
 		if(CLAN_MALKAVIAN)
 			my_clan = CLAN_MALKAVIAN
-			to_chat(owner, "<span class='announce'>You have Ranked up enough to learn: You are part of the Malkavian Clan!<br> \
-				* As part of the Malkavian Clan, you see the world in a different way, suffering hallucinations and seeing strange portals everywhere.</span>")
-			// WILLARD TODO: Make Masquerade hide brain traumas? Unless you're a Beefman, that is. Also applies to Frenzy.
+			to_chat(owner, "<span class='reallybig hypnophrase'>Welcome to the Malkavian...</span>")
+			to_chat(owner, "<span class='userdanger'>* Bloodsucker Malkavian: Vampire is you are completely and irrati-- unrepairably Insane...</span>")
 			bloodsucker.gain_trauma(/datum/brain_trauma/mild/hallucinations, TRAUMA_RESILIENCE_ABSOLUTE)
 			bloodsucker.gain_trauma(/datum/brain_trauma/special/bluespace_prophet, TRAUMA_RESILIENCE_ABSOLUTE)
+			ADD_TRAIT(bloodsucker, TRAIT_XRAY_VISION, BLOODSUCKER_TRAIT)
```

Comment:
how does the night vision from bloodsucker eyes interact with x-ray? we're sure that you can see in the dark through walls, right?

Reply:
You can. It's complete x-ray with no darkness whatsoever, thats intended.